### DISCUSSION TITLE
Guacamole LXC

### DISCUSSION THREAD

**Discussion by FabioEight** [tteck/Proxmox#1081] [Ideas]
Hello,

I would suggest to support the installation of Guacamole in a LXC, it is really useful for access the GUI of other hosts/guest.

Cheers!

**Comment by NorthernMunkey** [comment]
agreed, i've seen this as a docker, not sure how much work it would take to port it over to lxc

**Reply by TheRealFu** [reply]
Guacamole is the sole reason I have Docker running right now.  I'd love to move it to an lxc in ProxMox VE.

**Comment by rallegade** [comment]
I would love to have a script for this too

**Comment by Cryptoman1960** [comment]
Built my own container by following these steps here.

https://dannyda.com/2020/05/23/how-to-install-apache-guacamole-on-debian-ubuntu-the-easiest-way-clientless-remote-desktop-gateway/

**Comment by CrazyWolf13** [comment]
@tteck  Any specific reason, why this has not been created yet?

Should be quite straightforward: https://guacamole.apache.org/doc/0.9.2/gug/installing-guacamole.html

**Reply by JustAnotherPU** [reply]
would love to see a script for Guacamole

**Reply by Narkowii** [reply]
me too !

**Reply by bramsuurdje** [reply]
The source you added is extremely outdated.

I suggest just installing a blank [Debian 12 LXC](https://helper-scripts.com/scripts?id=32k9mmr9rbkrs8t) and then running this [Guacamole install script](https://github.com/itiligent/Guacamole-Install). It can't get any simpler than that.

**Comment by bramsuurdje** [comment]
I suggest just installing a blank [Debian 12 LXC](https://helper-scripts.com/scripts?id=32k9mmr9rbkrs8t) and then running this [Guacamole install script](https://github.com/itiligent/Guacamole-Install). It can't get any simpler than that.

**Reply by CrazyWolf13** [reply]
Looks promising! Thanks!

Pheww, don't know how google could infiltrate me that old version of the Docs...
